docs/guidance/map/awards_contracts_buyers_suppliers.md

```diff
@@ -36,55 +31,36 @@ Contracting processes can result in different types of contract between buyers a
 
 In OCDS, the `Contract` object is intended to communicate a legally binding agreement between a buyer and suppliers to provide items. This excludes agreements to set-up a structure through which contracts are later awarded to provide items, for example: a contract to set up or add suppliers to a framework agreement or dynamic purchasing system.
 
-```{eval-rst}
-.. admonition:: Note
-    :class: note
-
-    .. markdown::
-
-      The OCDS schema and documentation are not clear what, precisely, is meant by 'contract'. A revision of the definition of `Contract` is being considered for a future version of the standard ([GitHub issue](https://github.com/open-contracting/standard/issues/896)).
-
+```{note}
+The OCDS schema and documentation are not clear what, precisely, is meant by 'contract'. A revision of the definition of `Contract` is being considered for a future version of the standard ([GitHub issue](https://github.com/open-contracting/standard/issues/896)).
 ```
 
 ### Buyer
 
 OCDS defines the buyer as:
 
-```{eval-rst}
-.. field-description:: ../../../build/current_lang/release-schema.json /properties/buyer
+```{field-description} ../../../build/current_lang/release-schema.json /properties/buyer
 ```
 
-```{eval-rst}
-.. admonition:: Note
-    :class: note
+```{note}
+Elsewhere in the OCDS schema the buyer is described as:
 
-    .. markdown::
-
-      Elsewhere in the OCDS schema the buyer is described as:
-
-      > *The entity managing the procurement. This may be different from the buyer who pays for, **or uses**, the items being procured.*
-
-      There is a proposal to align this description with the above definition.
+> *The entity managing the procurement. This may be different from the buyer who pays for, **or uses**, the items being procured.*
 
+There is a proposal to align this description with the above definition.
 ```
 
 ### Suppliers
 
 OCDS defines a supplier as:
 
-```{eval-rst}
-.. code-description:: ../../../build/current_lang/codelists/partyRole.csv supplier
+```{code-description} ../../../build/current_lang/codelists/partyRole.csv supplier
 ```
 
 ### Electronic catalog
 
-```{eval-rst}
-.. admonition:: Hint
-    :class: hint
-
-    .. markdown::
-
-      Electronic catalogs can often be found in more complicated procedures, usually together with framework agreements and involving multiple bidders. However, on their own, they are simply an electronic format (typically prescribed by the buyer) that participants in the contracting process must follow when exchaning information about technical specifications, evaluation criteria, bids, lots, etc. As such, catalogs do not influence the relationship between awards, contracts, buyers and suppliers. 
+```{note}
```

Comment:
The original was a hint admonition. On the other hand, I think this content is fine as a note.